charts/argo-cd/templates/NOTES.txt

```diff
@@ -1,3 +1,8 @@
+{{- if .Values.configs.repositoryCredentials }}
+WARNING: You are using configs.repositoryCredentials parameter that's DEPRECATED
+Instead, use configs.repositoryTemplates and/or configs.repositories parameters
+
```

Comment:
What about print the url to read more about this? 
-> https://argo-cd.readthedocs.io/en/latest/operator-manual/declarative-setup/#legacy-behaviour